app/javascript/mastodon/locales/whitelist_en.json

```diff
@@ -1,2 +1 @@
-[
-]
+[]
```

Comment:
Unrelated, but I noticed these were mostly empty for the other locales as well, and wasn't sure if it was still being used